### PR TITLE
OHRM5X-2066: Develop oauth - consent screen FE

### PR DESCRIPTION
**Pull request by devishke-orange**
PR contains:
- OHRM5X-2066: [FE] Develop consent screen

Updated SS in a comment below

Mobile View
image

Mobile View - Invalid Request
image

Web View
image

Web View - Invalid Request
image

### CODE REVIEW

**Comment by devishke-orange** [comment]
Example Google OAuth Error Screens:
image

image

File: installer/Migration/V5_4_0/lang-string/auth.yaml [hunk 1]
```diff
@@ -32,3 +32,30 @@ langStrings:
   -
     value: 'Invalid password reset code'
     unitId: invalid_password_reset_code
+  -
+    value: '{clientName} would like to'
+    unitId: client_name_would_like_to
+  -
+    value: 'Access your basic profile information'
+    unitId: access_your_basic_profile_info
```
Comment (RajithaKumara) [inline installer/Migration/V5_4_0/lang-string/auth.yaml:40]: I feel this point already covered under `Access and manage your data`

File: installer/Migration/V5_4_0/lang-string/auth.yaml [hunk 2]
```diff
@@ -32,3 +32,30 @@ langStrings:
...
+  -
+    value: 'Access and manage your data'
+    unitId: access_and_manage_your_data
+  -
+    value: 'Perform requests on your behalf'
+    unitId: perform_requests_on_your_behalf
+  -
+    value: 'Keep you signed in to the app'
+    unitId: keep_you_signed_in_to_the_app
```
Comment (RajithaKumara) [inline installer/Migration/V5_4_0/lang-string/auth.yaml:49]: Will remove this

File: installer/Migration/V5_4_0/lang-string/auth.yaml [hunk 3]
```diff
@@ -32,3 +32,30 @@ langStrings:
...
+    unitId: client_name_would_like_to
+  -
+    value: 'Access your basic profile information'
+    unitId: access_your_basic_profile_info
+  -
+    value: 'Access and manage your data'
+    unitId: access_and_manage_your_data
+  -
+    value: 'Perform requests on your behalf'
+    unitId: perform_requests_on_your_behalf
```
Comment (RajithaKumara) [inline installer/Migration/V5_4_0/lang-string/auth.yaml:46]:
```suggestion
    value: 'Perform actions on your behalf'
    unitId: perform_actions_on_your_behalf
```
:thinking:

File: src/client/src/orangehrmCoreOAuthPlugin/pages/oauth/oauth-authorize.scss [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+//
+// OrangeHRM is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+// without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+// See the GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License along with this program;
+// if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+// Boston, MA  02110-1301, USA
+
+.orangehrm-oauth {
```
Comment (RajithaKumara) [inline src/client/src/orangehrmCoreOAuthPlugin/pages/oauth/oauth-authorize.scss:17]: According to `Web View` SS, seems we can allocate more width for the card.
Hope the design similar to Maintenance Admin access and Forgot password screens

**Review by RajithaKumara — CHANGES_REQUESTED** [review]

**Comment by devishke-orange** [comment]
Updated SS:

Mobile View
image

Mobile View - Invalid Request
image

Web View
image

Web View - Invalid Request
image

**Pull request merged**